SECURITY.md

```diff
@@ -0,0 +1,49 @@
+# Security Policy
+
+## Supported Versions
+
+We actively maintain and provide security updates for the following versions:
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 3.x     | :white_check_mark: |
+| 2.x     | :x:                |
+
+## Known Security Issues
+
+### Modernizr 2.8.3 and Earlier
+
+**Issue**: Dynamic code execution via `Function()` constructor
+**Severity**: Medium
+**Status**: Fixed in 3.x
+
+Modernizr versions 2.8.3 and earlier contain code patterns that use the `Function()` constructor for dynamic code execution, which can trigger security scanners like SonarQube.
+
+**Example problematic code**:
+```javascript
+ownerDocument.createDocumentFragment = Function('h,f', 'return function(){');
+```
+
+**Recommendation**: Upgrade to Modernizr 3.x which removes these patterns and uses safer alternatives.
+
+## Reporting a Vulnerability
+
+If you discover a security vulnerability, please report it by:
+
+1. **Do not** open a public issue
+2. Email the maintainers directly
```

Comment:
Not sure if the contact form on https://modernizr.com/resources/
is still good.
Maybe @paulirish can say if https://paulirish.wufoo.com/forms/p1hsxsru1ohiv0u/ is still monitored in any way?

Reply:
@1234-ad please replace step 2 with a link to the contact form from the modernizr homepage. if you got questions, please ask directly and not an AI, that would be very helpful to both of us.